src/main/java/org/mongojack/JacksonDBCollection.java

```diff
@@ -2178,6 +2178,43 @@ public DBObject convertToDbObject(T object) throws MongoException {
     }
 
     /**
+     * This method provides a static way to convert an object into a DBObject. Defaults will be used for all parameters
+     * left null.
+     * 
+     * @param object The object to convert
+     * @param useStreamSerialization Whether to use stream Serialization. (Default false)
+     * @param view The Jackson View to use in serialization. (Default null)
+     * @param objectMapper The specific Jackson ObjectMapper to use. (Default MongoJack ObjectMapper)
+     * @return
+     */
+    public static <T> DBObject convertToDbObject(T object, Boolean useStreamSerialization, Class<?> view, ObjectMapper objectMapper) {
```

Comment:
can we make the existing `convertToDbObject` method call this one?

Reply:
👍